app/Transformers/TaskSchedulerTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+
+namespace App\Transformers;
+
+
+use App\Models\ScheduledJob;
+use App\Models\Scheduler;
+use App\Utils\Traits\MakesHash;
+
+class TaskSchedulerTransformer extends EntityTransformer
+{
+    use MakesHash;
+
+    protected $defaultIncludes = [
+        'job'
+    ];
+
+    public function includeJob(Scheduler $scheduler)
+    {
+        $transformer = new ScheduledJobTransformer($this->serializer);
+
+        return $this->item($scheduler->job, $transformer, ScheduledJob::class);
+    }
+
+    public function transform(Scheduler $scheduler)
+    {
+        return [
+            'id' => $this->encodePrimaryKey($scheduler->id),
+            'company_id' => $this->encodePrimaryKey($scheduler->user_id),
+            'paused' => $scheduler->paused,
+            'archived' => $scheduler->archived,
```

Comment:
please review other transformers, we cast all timestamps to int also review the naming structure of the timestamps in the transformer, ie

```
            'updated_at' => (int) $tax_rate->updated_at,
            'archived_at' => (int) $tax_rate->deleted_at,
            'created_at' => (int) $tax_rate->created_at,
```